components/Map.js

```diff
@@ -354,7 +354,7 @@ function Map({ adminAddress, adminContract, account, ceramic, ipfs }) {
           ceramic={ceramic}
           ipfs={ipfs}
         ></Sidebar>
-      ) : null}
+      ) : null}gridLayer
```

Comment:
I think this is the cause. Is this here for any particular reason or just left over from debugging?

Reply:
Removing `gridLayer` locally fixed the scrolling/whitespace issue for me.

Reply:
I think some IDE fault.. while debugging.